Samples/WinMLSamplesGallery/TransformAsync/SegmentModel.cpp

```diff
@@ -0,0 +1,279 @@
+}
+
+LearningModel BackgroundBlur::PostProcess(long n, long c, long h, long w, long axis)
+{
+	auto builder = LearningModelBuilder::Create(opset)
+		.Inputs().Add(LearningModelBuilder::CreateTensorFeatureDescriptor(L"InputImage", TensorKind::Float, { n, c, h, w }))
+		.Inputs().Add(LearningModelBuilder::CreateTensorFeatureDescriptor(L"InputScores", TensorKind::Float, { -1, -1, h, w })) // Different input type? 
+		.Outputs().Add(LearningModelBuilder::CreateTensorFeatureDescriptor(L"OutputImage", TensorKind::Float, { n, c, h, w }))
+		// Argmax Model Outputs
+		.Operators().Add(LearningModelOperator(L"ArgMax")
+			.SetInput(L"data", L"InputScores")
+			.SetAttribute(L"keepdims", TensorInt64Bit::CreateFromArray({ 1 }, { 1 }))
+			.SetAttribute(L"axis", TensorInt64Bit::CreateFromIterable({ 1 }, { axis })) // Correct way of passing axis? 
+			.SetOutput(L"reduced", L"Reduced"))
+		.Operators().Add(LearningModelOperator(L"Cast")
+			.SetInput(L"input", L"Reduced")
+			.SetAttribute(L"to", TensorInt64Bit::CreateFromIterable({}, { OnnxDataType::ONNX_FLOAT }))
+			.SetOutput(L"output", L"ArgmaxOutput"))
+
+		// Extract the foreground using the argmax scores to create a mask
+		.Operators().Add(LearningModelOperator(L"Clip")
+			.SetInput(L"input", L"ArgmaxOutput")
+			.SetConstant(L"min", TensorFloat::CreateFromIterable({ 1 }, { 0.f }))
+			.SetConstant(L"max", TensorFloat::CreateFromIterable({ 1 }, { 1.f }))
+			.SetOutput(L"output", L"MaskBinary"))
+		.Operators().Add(LearningModelOperator(L"Mul")
+			.SetInput(L"A", L"InputImage")
+			.SetInput(L"B", L"MaskBinary")
+			.SetOutput(L"C", L"ForegroundImage"))
+
+		// Extract the blurred background using the negation of the foreground mask
+		.Operators().Add(LearningModelOperator(L"AveragePool") // AveragePool to create blurred background
+			.SetInput(L"X", L"InputImage")
+			.SetAttribute(L"kernel_shape", TensorInt64Bit::CreateFromArray(std::vector<int64_t>{2}, std::array<int64_t, 2>{20, 20}))
+			.SetAttribute(L"auto_pad", TensorString::CreateFromArray(std::vector<int64_t>{1}, std::array<hstring, 1>{L"SAME_UPPER"}))
+			.SetOutput(L"Y", L"BlurredImage"))
+		.Operators().Add(LearningModelOperator(L"Mul")
+			.SetInput(L"A", L"MaskBinary")
+			.SetConstant(L"B", TensorFloat::CreateFromIterable({ 1 }, { -1.f }))
+			.SetOutput(L"C", L"NegMask"))
+		.Operators().Add(LearningModelOperator(L"Add") // BackgroundMask = (1- foreground Mask)
+			.SetConstant(L"A", TensorFloat::CreateFromIterable({ 1 }, { 1.f }))
+			.SetInput(L"B", L"NegMask")
+			.SetOutput(L"C", L"BackgroundMask"))
+		.Operators().Add(LearningModelOperator(L"Mul") // Extract the blurred background
+			.SetInput(L"A", L"BlurredImage")
+			.SetInput(L"B", L"BackgroundMask")
+			.SetOutput(L"C", L"BackgroundImage"))
+
+		// Combine foreground and background
+		.Operators().Add(LearningModelOperator(L"Add")
+			.SetInput(L"A", L"ForegroundImage")
+			.SetInput(L"B", L"BackgroundImage")
+			.SetOutput(L"C", L"OutputImage"))
+		;
```

Comment:
>; [](http://example.com/codeflow?start=2&length=1)

[nit] At first I was going to say something about this dangling semicolon, but then I realized it's useful for diffs to more clearly see insertions and deletions (because the semicolon doesn't get in the way). So I'll say nothing here, but it does look funny comparing line 195 (indented on next line) vs 253 (on same line) vs 276 (not indented on next line)